tests/best-practices/kyverno-test.yaml

```diff
@@ -0,0 +1,117 @@
+name: check-amazon-inspector
```

Comment:
It would be good to have multiple negative cases. This is probably fine for this repo. But when we write tests for the curated policies, our tests should be more comprehensive.

Reply:
As per the discussion on slack, more comprehensive tests will be added in the [kyverno-policies](https://github.com/nirmata/kyverno-policies/) repo.